src/terminfo/mod.rs

```diff
@@ -114,11 +113,11 @@ impl TermInfo {
             Ok(TermInfo {
                 names: vec![name.to_owned()],
                 bools: HashMap::new(),
-                numbers: numbers,
-                strings: strings,
+                numbers,
+                strings,
```

Comment:
Let's not do this: https://github.com/rust-lang/rust-clippy/issues/3422. I'd rather just disable the lint.